### DISCUSSION TITLE
Spring Core

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#40] [General]
# Spring Core

4/11 레벨2를 슬기롭게 소화하기 학습해야하는 내용의 범위가 넓으니, 깊이를 제한한다. 얼마까지 해야 할까? -> 프롤로그 참고하기 다른 웹 서비스를 유심히 살펴보자. 평소 자주 사용하거나 내가 만드는 것과 비슷한 서비스를 뜯어본다. 현재 상태에 …

https://cyma-s.github.io/spring-core/

**Comment by kevstevie** [comment]
세터 주입 쓰지마세요
